src/endToEnd.ts

```diff
@@ -1,20 +1,24 @@
-import { VotesDict } from '@votingworks/ballot-encoder'
+import { CompletedBallot } from '@votingworks/ballot-encoder'
 
 import { E2EAPI } from './config/types'
 import fetchJSON from './utils/fetchJSON'
 
-export default async function encryptBallot(
-  // eslint-disable-next-line
-  votes: VotesDict
-) {
+export default async function encryptBallot(ballot: CompletedBallot) {
   try {
     const { tracker } = await fetchJSON<E2EAPI>(
       '/electionguard/EncryptBallot',
       {
         method: 'post',
-        body: JSON.stringify(votes),
+        body: JSON.stringify({
+          ballot: ballot,
+          currentBallotCount: 0,
```

Comment:
What is this?

Reply:
this is the total number of currently cast ballots, for sync'ing with EG backend. I'm setting it to 0 for now as I think it's unused, but will improve this.